modules/services/images/lint-baseline.xml

```diff
@@ -752,14 +752,6 @@
         message="One or more images in this project can be converted to the WebP format which typically results in smaller file sizes, even for lossless conversion">
         <location
             file="src/main/res/drawable-xxxhdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-xxhdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-xxhdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-mdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-hdpi/appicon_patron_dark.png"/>
```

Comment:
This is removed by the lint update baseline, or manually?

Reply:
When generating. And it doesn't fail now unlike previous times: https://buildkite.com/automattic/pocket-casts-android/builds/6741#018fc355-4cd2-4c4b-909a-586ad829a3e0

Reply:
Oh wow… it is sooo weird. @ashiagr generated baseline rules in her PR as well [here](https://github.com/Automattic/pocket-casts-android/pull/2279/commits/b2f7baf79862c14158fb5d765e503078a35ec111). And it is different from what is generated on my machine. I've checked out her branch and generated again and this is the diff.

[baseline.patch](https://github.com/Automattic/pocket-casts-android/files/15485200/baseline.patch)